docs/content/en/usage/features.md

```diff
@@ -5,6 +5,142 @@ draft: false
+     - Finding Will *not* be returned
+       - Finding Tags: ["A", "B", "C"]
+       - Filter Query: "B"
+ - Tag Name Contains: filter on any tags conatin part or all of the query in the given Finding
+   - Exmaples:
+     - Finding will be returned
+       - Finding Tags: ["Alpha", "Beta", "Charlie"]
+       - Filter Query: "et" (part of "Beta")
+     - Finding Will *not* be returned
+       -  Finding Tags: ["Alpha", "Beta", "Charlie"]
+       - Filter Query: "meg" (part of "Omega")
+ - Not Tags: filter on any tags that do *not* conatin part or all of the query in the given Finding
+   - Exmaples:
+     - Finding will be returned
+       -  Finding Tags: ["Alpha", "Beta", "Charlie"]
+        Filter Query: "meg" (part of "Omega")
+     - Finding Will *not* be returned
+       -  Finding Tags: ["Alpha", "Beta", "Charlie"]
+       - Filter Query: "et" (part of "Beta")
+
+For the other six tag filters, they follow the same rules as "Tags" and "Not Tags" as above,
+but at different levels in the data model:
+
+ - Tags (Test): filter on any tags that are attached to the Test of a given Finding is apart of
+ - Not Tags (Test): filter on any tags that are *not* attached to the Test of a given Finding is apart  of
+ - Tags (Engagement): filter on any tags that are attached to the Engagement of a given Finding is apart of
+ - Not Tags (Engagement): filter on any tags that are *not* attached to the Engagement of a given Finding is apart  of
+ - Tags (Product): filter on any tags that are attached to the Product of a given Finding is apart of
+ - Not Tags (Product): filter on any tags that are *not* attached to the Product of a given Finding is apart  of
+
+### Tag Inheritance
+
+When enabled, tags applied to a given product will automatically be applied to all objects under 
+products in the [data model](../models.md).
+
+#### Configuration
+
+Tag Inheritance can be enabled at the following scope levels:
+- Global Scope
+  - Every product system wide will begin applying tags to all children objects
+  - This is set within the System Settings
+- Product Scope
+  - Only the selected product will begin applying tags to all children objects
+  - This is set at the product creation/edit page
+
+#### Behaviors
+
+Tags can be added and removed to other objects the same as when tag inheritance
+is disabled. The only exception to that rule being inherited tags as they cannot
+be removed from an object. See the following example of adding a tag "test_only_tag"
+to the Test obejct and a tag "engagement_only_tag" to the Engagement.
+
+![Example of inherited tags](../../images/tags-inherit-exmaple.png)
+
+When updates are made to the tag list on a product, the same changes are made to all
+objects within the product asyncronously. The duration of this task directly correlates
+to the number the objects contained within a finding. If the results are not observed
+within a reasonable amount, consult to the celery worker logs to identify where any problems
```

Comment:
Nit: s/.../within a reasonable time period, consult the celery worker logs to identify where any problems might have arisen/